zone/zone.cpp

```diff
@@ -158,6 +158,8 @@ bool Zone::Bootup(uint32 iZoneID, uint32 iInstanceID, bool is_static) {
 	LogInfo("Zone bootup type [{}] short_name [{}] zone_id [{}] instance_id [{}]",
 		(is_static) ? "Static" : "Dynamic", zonename, iZoneID, iInstanceID);
 	parse->Init();
+	parse->ReloadQuests();
+	zone->Repop();
```

Comment:
The call to `parse->Init()` already calls ReloadQuests for Lua so this would unncessarily reload quests twice. It does not currently call it for Perl since https://github.com/EQEmu/Server/pull/3648 was backed out for breaking something with perl (I never followed up on the report, it may be a similar regression). This would also essentially repop twice since Repop does the same things as  zone->Init. 

The original issue with zone controller on EZ Server is likely caused by a regression in #4025 which added calls to check `zone->IsLoaded` in places that it previously didn't. Even before `parse->Init()` is called, the call to `zone->Init` triggers quest events for zone controller. These dispatches were disabled with that patch since `is_zone_loaded` is set to true only after the call to `zone->Init`.